Let eclipse-wtp remove the "jst.utility" facet in ejb modules (#17699)
gradle/gradle: subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

To ignore directories in Gradle 8.0 and later, the input property needs to be explicitly annotated with `@IgnoreEmptyDirectories`.
For more information see <<empty_directories_file_tree>>.

> ==== Project facets generated by the `eclipse wtp' plugin have changed

Reviewer: Short and clear. 👍 for that!

> ==== Project facets generated by the `eclipse wtp' plugin have changed

Reviewer 2: ```suggestion
==== Adding `jst.ejb` with the `eclipse wtp' plugin now removes the `jst.utility` facet
```
Suggestion: the proposed title is short, but I think it could be clearer. Perhaps a reword can provide additional detail without getting too lengthy.

> A `jst.utility` facet was added to java projects by the `eclipse wtp` plugin. 
> This facet will now be removed when you add a `jst.ejb` facet: 

Reviewer 2: ```suggestion
The `eclipse wtp` plugin adds the `jst.utility` facet to java projects.
Now, adding the `jst.ejb` facet implicitly removes the `jst.utility` facet: 
```
Nitpick: use active voice and simpler phrasing when possible.
Suggestion: it might be useful for readers to know why we avoid combining these facets. Consider adding a glue sentence in between that explains that motivation -- "But using the `jst.ejb` and `jst.utility` facets in the same projects led to _____." -- or similar.
